flamedisx/lxe_blocks/detection.py

```diff
@@ -123,13 +133,17 @@ def _compute(self, data_tensor, ptensor,
 class DetectElectrons(DetectPhotonsOrElectrons):
     dimensions = ('electrons_produced', 'electrons_detected')
 
-    special_model_functions = ('electron_acceptance',)
+    special_model_functions = ('electron_acceptance', 'extraction_eff')
     model_functions = ('electron_detection_eff',) + special_model_functions
 
     @staticmethod
     def electron_detection_eff(drift_time, *,
```

Comment:
Maybe you could rename electron_detection_eff to electron_drift_eff, to emphasize the extraction efficiency is no longer part of it?

Reply:
Ah you're right, since this class handles both photons and electrons that would indeed require extra if statements. Never mind.